ledger/complete/wal/checkpointer.go

```diff
@@ -293,7 +286,18 @@ func CreateCheckpointWriterForFile(dir, filename string, logger *zerolog.Logger)
 // as for each node, the children have been previously encountered.
 // TODO: evaluate alternatives to CRC32 since checkpoint file is many GB in size.
 // TODO: add concurrency if the performance gains are enough to offset complexity.
-func StoreCheckpoint(writer io.Writer, tries ...*trie.MTrie) error {
+func StoreCheckpoint(dir string, fileName string, logger *zerolog.Logger, tries ...*trie.MTrie) error {
```

Comment:
checkpoint V6 will start using dir and fileName, changing it to make the transition easier